rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly-2022-11-28
```

Comment:
needed? or else too opinionated?

Reply:
I'd avoid nightly personally if at all possible (but perhaps it's not in this case; I haven't dug into the code yet). Assuming it's needed, I think pinning to a specific one is a good approach to avoid random breakages and such (like any version it should periodically be updated though, so perhaps a more recent one is possible?)